### PR TITLE
Added seperate SMTP login username and password

### PR DESCRIPTION
**Pull request by chirmstream**
I added smtp_username and smtp_password options in case your outgoing email goes through an smtp server with different login credentials than your imap server.

### CODE REVIEW

**Comment by rob-luke** [comment]
@dglb99 I just pushed a change directly to your branch. Can you test if this works locally and report back? You will need to do a 'git pull origin master' (I think you are using master branch) to receive my latest changes.

**Comment by dglb99** [comment]
@rob-luke, I just tried your changes and they work locally for me.

File: src/main.py [hunk 1]
```diff
@@ -235,6 +237,9 @@ def _get_imap_filter(mail_message_flag):
     password = os.environ.get("IMAP_PASSWORD")
     folder = os.environ.get("IMAP_FOLDER")
 
+    smtp_username = os.environ.get("SMTP_USERNAME")
```
Comment (rob-luke) [inline src/main.py:240]:
```suggestion
    smtp_username = os.environ.get("SMTP_USERNAME", username)
```

File: src/main.py [hunk 2]
```diff
@@ -235,6 +237,9 @@ def _get_imap_filter(mail_message_flag):
...
+    smtp_password = os.environ.get("SMTP_PASSWORD")
```
Comment (rob-luke) [inline src/main.py:241]:
```suggestion
    smtp_password = os.environ.get("SMTP_PASSWORD", password)
```

**Review by rob-luke — COMMENTED** [review]
For backwards compatability we should use the previous value as a defaiult

**Pull request merged**